sentry-spring-boot-starter/src/test/kotlin/io/sentry/spring/boot/SentryLogbackAppenderAutoConfigurationTest.kt

```diff
@@ -29,16 +29,24 @@ class SentryLogbackAppenderAutoConfigurationTest {
     }
 
     @Test
-    fun `configures SentryAppender`() {
+    fun `hub is not created when auto-configuration dsn is not set`() {
```

Comment:
I think this test should verify that "does not configure SentryAppender when auto-configuration dsn is not set".

Reply:
Improved, thanks.